Add to README hints how to debug failures in CI (#2605)
openSUSE/openSUSE-release-tools: README.md

As mentioned before, the main repository uses GitHub Actions to automatically run the tests when a pull request is opened or the code is pushed to the master branch. You can find the details in the
[workflow definition](.github/workflows/ci-test.yml). Note that, in addition to the steps listed before, code coverage data is submitted to [Codecov](https://app.codecov.io/gh/openSUSE/openSUSE-release-tools).

### Debugging Failures in CI

The first step is to reproduce it when running it locally as described above.
To see all logs following command can be executed:

  docker-compose -f dist/ci/docker-compose.yml logs -f --tail=10

> To check content of any running container use this command ( in this case "api" container ) to get shell:

Reviewer: ```suggestion
You can run commands in any container by using the docker-compose `exec` command. For instance, you can connect to a container through a shell with the following command (in this case, it will connect to the container behind the `api` service):
```

  docker-compose -f dist/ci/docker-compose.yml exec api sh

> 

Reviewer: ```suggestion
Or you could check the API logs by issuing the following command:

  docker-compose -f dist/ci/docker-compose.yml exec api sh -c 'tail -f /srv/www/obs/api/log/*.log'
```

> Also OBS in container can be reached at `http://0.0.0.0:3000`. There are user "Admin" with password "opensuse".

Reviewer: ```suggestion
You can access OBS at `http://0.0.0.0:3000` and log in using ' Admin'  as username and `opensuse` as password.
```

Author: why additional space before Admin and having Admin in '' and opensuse in different ticks, looks a bit strange for me.

Reviewer: Sorry, a typo.

> To prevent cleaning in testsuite place `breakpoint()` in code to stop execution and drop to debugger.

Reviewer 2: btw: I usually add a `sleep 30000` - breakpoints are too modern for me :)

> To prevent cleaning in testsuite place `breakpoint()` in code to stop execution and drop to debugger.

Reviewer: ```suggestion
To debug problems in the test suite or in the code, place a `breakpoint()` call and you will get access to Python's debugger.
```

Author: Well, goal of this line is to add hint how to check in OBS web ui what is status before test clean up state and destroy projects. So this change a kind of kill that intention.

Reviewer: OK, I will clarify a little bit.
